package.json

```diff
@@ -8,16 +8,20 @@
     "build": "vite build",
     "preview": "vite preview",
     "fmt": "npx prettier --write *.html *.js *.svelte components/*",
-    "lint": "npx eslint . --ignore-pattern=vite.config.js"
+    "lint": "npx eslint . --ignore-pattern=vite.config.js",
+    "test": "vite build && vitest"
```

Comment:
`build` takes a few seconds... but I think this default makes sense. If you're working on the test code and not ATIP and don't trust the auto-rebuild, `npx vitest` works too.

Reply:
That's a good spot and saves time for me locally!